go/coordinator/internal/coordinator/apis_test.go

```diff
@@ -893,42 +896,45 @@ func TestUpdateSegment(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, []*model.Segment{segment}, result)
 
+	// TODO: revisit why we need this
 	// Update collection to new value
-	segment.CollectionID = sampleCollections[1].ID
```

Comment:
Why comment these tests? These tests are ported from the python system test.

Reply:
I don't see why we need to set collection id to null or modify the collection id for a segment. And it is not possible to do it because collection_id is now primary key of segment